Source/compositorclient/cmake/FindWesterosClient.cmake

```diff
@@ -0,0 +1,70 @@
+# - Try to find BroadCom nxserver library.
+# Once done this will define
+#  WESTEROS_FOUND - System has westeros
+#  WESTEROS_INCLUDE_DIRS - The westeros include directories
```

Comment:
same as above, missing CLIENT in the name

Reply:
added